packages/eightballer/skills/simple_fsm/behaviours.py

```diff
@@ -79,6 +94,7 @@ def act(self) -> None:
             not self.strategy.state.bridge_requests
             and not self.strategy.state.bridging_in_progress
             and not self.strategy.state.waiting_balance_difference
+            and not True
```

Comment:
this is odd, likely was a quick hack @8ball030 that needs to be removed? or what's the intention of this